modular_skyrat/modules/event_vote/code/event_vote.dm

```diff
@@ -427,27 +429,27 @@
 			register_vote(usr, selected_event)
 			return
 		if("end_vote")
-			if(!check_rights(R_PERMISSIONS))
+			if(!check_rights(R_FUN))
 				return
 			end_vote(usr)
 			return
 		if("cancel_vote")
-			if(!check_rights(R_PERMISSIONS))
+			if(!check_rights(R_FUN))
 				return
 			cancel_vote(usr)
 			return
 		if("start_vote_admin")
-			if(!check_rights(R_PERMISSIONS))
+			if(!check_rights(R_FUN))
 				return
 			start_vote_admin()
 			return
 		if("start_vote_admin_chaos")
-			if(!check_rights(R_PERMISSIONS))
+			if(!check_rights(R_FUN))
 				return
 			start_vote_admin_chaos()
 			return
 		if("start_player_vote")
-			if(!check_rights(R_PERMISSIONS))
+			if(!check_rights(R_FUN))
 				return
```

Comment:
At the very least, I want these four (so including the `start_player_vote_chaos`) to remain based on `R_PERMISSIONS`, as we specifically requested for those to be usable by Staff Managers and above only.

If you want that to be based on a config or whatever, sure, that's fine by me, but as it stands for Skyrat we want those permissions to be for SM+, as they shouldn't really ever be ran in the current state of things anyway.